metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -541,6 +575,64 @@ static void validateNewTopicNames(Map<String, ApiError> topicErrors,
         return configChanges;
     }
 
+    Map<String, ResultOrError<Uuid>> findTopicIds(long offset, Collection<String> names) {
+        Map<String, ResultOrError<Uuid>> results = new HashMap<>();
```

Comment:
nit: I guess we could initialize the size (a couple similar cases below as well)